apps/vaporgui/MainForm.cpp

```diff
@@ -642,12 +645,14 @@ void MainForm::_createAnimationToolBar()
 
     _animationToolBar->setWhatsThis(qat);
 
-    connect(_playForwardAction, SIGNAL(triggered()), _tabMgr, SLOT(AnimationPlayForward()));
-    connect(_playBackwardAction, SIGNAL(triggered()), _tabMgr, SLOT(AnimationPlayBackward()));
-    connect(_pauseAction, SIGNAL(triggered()), _tabMgr, SLOT(AnimationPause()));
-    connect(_stepForwardAction, SIGNAL(triggered()), _tabMgr, SLOT(AnimationStepForward()));
-    connect(_stepBackAction, SIGNAL(triggered()), _tabMgr, SLOT(AnimationStepBackward()));
-    connect(_timeStepEdit, SIGNAL(returnPressed()), this, SLOT(_setTimeStep()));
+    // clang-format off
+    connect(_playForwardAction,  SIGNAL(triggered()),     _animationController, SLOT(AnimationPlayForward()));
+    connect(_playBackwardAction, SIGNAL(triggered()),     _animationController, SLOT(AnimationPlayReverse()));
+    connect(_pauseAction,        SIGNAL(triggered()),     _animationController, SLOT(AnimationPause()));
+    connect(_stepForwardAction,  SIGNAL(triggered()),     _animationController, SLOT(AnimationStepForward()));
+    connect(_stepBackAction,     SIGNAL(triggered()),     _animationController, SLOT(AnimationStepReverse()));
+    connect(_timeStepEdit,       SIGNAL(returnPressed()), this, SLOT(_setTimeStep()));
+    // clang-format on
```

Comment:
It's a shame that clang-format doesn't seem to have an option that lets us align our code like this.  Just commenting.  If there's an option that I've missed, maybe we can consider using it.